crates/control/src/motion/head_motion.rs

```diff
@@ -36,6 +37,7 @@ pub struct CycleContext {
     cycle_time: Input<CycleTime, "cycle_time">,
     has_ground_contact: Input<bool, "has_ground_contact">,
     motion_selection: Input<MotionSelection, "motion_selection">,
+    world_state: Input<WorldState, "world_state">,
```

Comment:
I don't like the practice of using the world state as an input outside of behavior, the interface is way too wide.
Currently only the look_at motion also does this, even though it only needs a ground to field.
You only need the role here, so use that directly.

In general, I think motion information should only be carried inside the MotionCommand, but that needs a larger refactor I think.